test/index.js

```diff
@@ -1278,11 +1285,16 @@ describe('list', function() {
     });
 
     it('returns a Nothing if the element is not found', function() {
-      eq(S.indexOf(10, [1, 2, 3, 4, 5]), S.Nothing());
+      eq(S.indexOf('x', ['b', 'a', 'n', 'a', 'n', 'a']), S.Nothing());
     });
 
     it('returns Just the index of the element found', function() {
-      eq(S.indexOf(3, [1, 2, 3, 4, 5]), S.Just(2));
+      eq(S.indexOf('a', ['b', 'a', 'n', 'a', 'n', 'a']), S.Just(1));
```

Comment:
This is a slight improvement as it asserts that Just the index of the *first* occurrence is returned. The other similar assertions are all fine, but I updated them to match.